src/Avalonia.Controls.ColorPicker/Themes/Fluent.xaml

```diff
@@ -0,0 +1,135 @@
+<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        x:CompileBindings="True"
+        xmlns:converters="using:Avalonia.Controls.Converters"
+        xmlns:primitive="using:Avalonia.Controls.Primitives">
+
+  <Styles.Resources>
+    <converters:EnumValueEqualsConverter x:Key="EnumValueEquals" />
+    <converters:CornerRadiusToDoubleConverter x:Key="TopLeftCornerRadius" Corner="TopLeft" />
+    <converters:CornerRadiusToDoubleConverter x:Key="BottomRightCornerRadius" Corner="BottomRight" />
+  </Styles.Resources>
+
+  <Style Selector="primitive|ColorSpectrum">
```

Comment:
@maxkatz6 Here is a stranger thing with namespaces. XamlIl will throw errors saying it can't find `ColorSpectrum` unless I add the namespaces everywhere. However, the DataGrid theme files don't require namespaces. How can I remove these?

Reply:
You need to add XmlnsDefinition attributes for namespaces used for ColorPicker components
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Controls.DataGrid/Properties/AssemblyInfo.cs#L7-L9

Reply:
Works great, thanks.